MatrixSDK/Data/MXRoomMember.m

```diff
@@ -68,11 +68,18 @@ - (instancetype)initWithMXEvent:(MXEvent*)roomMemberEvent
             // The user who made the last membership change is the event user id
             _originUserId = roomMemberEvent.sender;
             
-            // If defined, keep the previous membership information
             if (roomMemberEvent.prevContent)
             {
                 MXRoomMemberEventContent *roomMemberPrevContent = [MXRoomMemberEventContent modelFromJSON:roomMemberEvent.prevContent];
+                // If defined, keep the previous membership information
                 _prevMembership = [MXTools membership:roomMemberPrevContent.membership];
+                // Also take displayname and avatarUrl from prevContent if not already set
+                if (!_displayname) {
+                    _displayname = roomMemberPrevContent.displayname;
+                }
+                if(!_avatarUrl) {
+                    _avatarUrl = roomMemberPrevContent.avatarUrl;
```

Comment:
Hrmm yea I might need to check that. Maybe we check that the current membership is `leave` so that we are taking the last known displayname and avatar url for users that left?